sdk/ml/azure-ai-ml/azure/ai/ml/entities/_assets/_artifacts/_package/model_package.py

```diff
@@ -6,46 +6,38 @@
 
 from os import PathLike
 from pathlib import Path
-from typing import IO, AnyStr, List, Dict, Optional, Union
-
-
-from azure.ai.ml._restclient.v2023_02_01_preview.models import (
-    PackageRequest,
-    PackageResponse,
-    ModelPackageInput as RestModelPackageInput,
-    PackageInputPathId as RestPackageInputPathId,
-    PackageInputPathVersion as RestPackageInputPathVersion,
-    PackageInputPathUrl as RestPackageInputPathUrl,
-    CodeConfiguration,
-)
-
-from azure.ai.ml.entities._resource import Resource
+from typing import IO, AnyStr, Dict, List, Optional, Union
+
+from azure.ai.ml._restclient.v2023_02_01_preview.models import CodeConfiguration
+from azure.ai.ml._restclient.v2023_02_01_preview.models import ModelPackageInput as RestModelPackageInput
```

Comment:
Why are we refactoring all these imports? Is the "import (list of packages)" not preferred for some reason?